### ISSUE TITLE
Each time I login in, the daily duty window moves to the left and clips behind my UI

### ISSUE THREAD

**Issue by ImmyOdh** [MidoriKami/DailyDuty#146]
Every time I log into the game my Daily Duty window is sat in this position, clipped behind the UI, even after I move it back to the correct position. The normal position is about 50 units to the right. It only moves along the horizontal axis.

![1123123](https://user-images.githubusercontent.com/132557105/236154610-47bf876f-baa2-4a89-81d8-073183421224.PNG)

**Comment by MidoriKami** [comment]
Please attach a copy of `todo.config.json` before launching the game, and another copy after you have logged in.

**Comment by ImmyOdh** [comment]
Where can I find that file?

**Comment by MidoriKami** [comment]
If the game screen is getting resized at any point that can cause elements to shift, there is nothing I can do about that if the coordinates aren't changing in the file.

**Comment by ImmyOdh** [comment]
After some further investigation, I believe this happens whenever a longer line of text in the daily duty ui element would cause the box to overlap with another UI element. It will shift the daily duty window to stop the overlap, then shift it back when the longer line of text is no longer there.

I just observed it happen with the squadron mission timer pushing the window behind my hotbar (as in the screenshot I posted originally) to stop it overlapping with my menu icons (which are just to the right), then saw it shift back to the proper spot as soon as I had completed that task and re-zoned.
